docs/DafnyRef/Types.md

```diff
@@ -2409,7 +2409,7 @@ A one-dimensional array of `n` `T` elements is created as follows:
 a := new T[n];
 ```
 The initial values of the array elements are arbitrary values of type
-`T`. They can also be initialized using a sequence display expression, as follows:
+`T`. They can also be initialized using a sequence display expression delimited by square brackets, as follows:
```

Comment:
I think 'delimited by square brackets' implies that multiple elements are separated by square brackets, like `1 [] 2 [] 3`. I'd use the term 'enclosed' here.

The suggestion I made earlier was largely based on this existing piece from the manual, which reads "A sequence can be formed using a sequence display expression, which is a possibly empty, ordered list of expressions enclosed in square brackets."

What do you have against saying "They can be initialized using an ordered list of expressions enclosed in square brackets" ?